pages/_app.js

```diff
@@ -1,21 +1,71 @@
 import GlobalStyle from "../styles";
 import { SWRConfig } from "swr";
 import Layout from "@/components/Layout/Layout";
+import { ThemeProvider } from "styled-components";
+
+const theme = {
+  colors: {
+    dark: "black",
+    light: "white",
+    neutral: "grey",
+    danger: "red",
+  },
+  fonts: {
+    main: "system-ui",
+    serif: "Cambria, Cochin, Georgia, Times, serif",
+  },
+  fontSize: {
+    default: "1rem",
+    small: "0.8rem",
+    large: "2rem",
+    xl: "3rem",
+  },
+  fontWeight: {
+    light: "200",
+    normal: "400",
+    bold: "700",
+  },
+  spacing: {
+    xs: "4px",
+    s: "8px",
+    m: "12px",
+    l: "20px",
+    xl: "28px",
+    xxl: "32px",
+    xxxl: "44px",
+  },
+  effects: {
+    boxShadow: "",
+    dropShadow: "",
+    linearGradient: "-webkit-linear-gradient(#e3f710, #ff0000)",
+    radialGradient:
+      "radial-gradient(circle, rgba(42, 42, 42, 1) 0%, rgba(13, 13, 13, 1) 100%)",
+  },
+  borders: {
+    radiusSmall: "0.75rem",
+    radiusMedium: "1.25rem",
+    radiusLarge: "6.25rem",
+    radiusRound: "50%",
+    strength: "2px",
+  },
```

Comment:
I'd suggest choosing one unit and stick with it but i could be wrong :)

Reply:
very good point. what type of unit would you recommend?

Reply:
that's a great question :D I honestly can't give you an educated tip, so maybe a coach could help :)